### PR TITLE
Add recipe for aiochclient

### PR DESCRIPTION
**Pull request by xhochy**
Checklist

- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example)
- [x] Source is from official source
- [x] Package does not vend other packages
- [x] Build number is 0
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there

### CODE REVIEW

**Comment by xhochy** [comment]
@conda-forge/staged-recipes This is ready for review/merge.

File: recipes/aiochclient/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+  version: "{{ version }}"
+
+source:
+  url: "https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz"
+  sha256: "4b686be9fa140e5c5fc3ff77062ca91e8b521d2df3efd31868f032f773befb13"
+
+build:
+  number: 0
+  skip: True  # [win]
+  skip: True  # [py<36]
```
Comment (scopatz) [inline recipes/aiochclient/meta.yaml]: you don't need this and the version constraints below

File: recipes/aiochclient/meta.yaml [hunk 2]
```diff
@@ -0,0 +1,43 @@
...
+  script: "{{ PYTHON }} -m pip install . -vv "
+
+requirements:
+  build:
+    - "{{ compiler('c') }}"
+  host:
+    - cython
+    - pip
+    - python >=3.6
```
Comment (chrisburr) [inline recipes/aiochclient/meta.yaml]: This version constraint should be removed as you have `skip`:
```suggestion
    - python
```
Constraining the version directly **_only_** works properly with `noarch: python` packages.

File: recipes/aiochclient/meta.yaml [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+  run:
+    - aiohttp >=3.0.1
+    - python >=3.6
```
Comment (chrisburr) [inline recipes/aiochclient/meta.yaml]: Similarly
```suggestion
    - python
```

**Pull request merged**